source/blog/2016-07-29-sprint-43-review-july-13-2016.html.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Sprint 43 review - July 13, 2016
+author: cybette
+date: 2016-07-29 14:38:08 UTC
+comments: true
+published: true
+---
+
+There is a ManageIQ Sprint review every 3 weeks, and everyone in the community is welcome to participate. To join our Sprint review meeting, please use this [Bluejeans link](https://bluejeans.com/946365937/).
+
+For Sprint 43, we merged 329 pull requests, making this a regular sprint averaging 100 PRs per week. There is a good distribution between new features, bug fixes, and code refactoring. We have plenty of enhancements on the Providers side, including Amazon, Networking, and Hawkular.  Work has started on Service Broker, which is a new piece for ManageIQ to allow VM selection based on criteria you provide.  
```

Comment:
@chriskacerguis Maybe you could write a blog post highlighting what you are working on in this area.